app/scrapers/thueringen_landtag_scraper.rb

```diff
@@ -78,6 +80,7 @@ def self.extract_paper(title_el)
     legislative_term, reference = extract_reference(full_reference)
     path = extract_path(title_el)
     url = extract_url(path)
+    return nil if UNSOPPORTED.include? extract_doctype_el(next_row).text.strip
```

Comment:
maybe its better to check the other way around - so check if its an major/minor interpellation and drop everything else